sesman/session.c

```diff
@@ -598,28 +598,48 @@ session_start_fork(tbus data, tui8 type, struct SCP_SESSION *s)
                 auth_set_env(data);
                 if (s->directory != 0)
                 {
-                    if (s->directory[0] != 0)
+                    if (g_strncasecmp("NO", g_cfg->sec.allowed_alternate_shells, 3) != 0)
```

Comment:
As you suspected, please use `g_text2bool()` here. If we want a list, we can then use a separate parameter. At the moment we're considering moving the config format to TOML, which means the separate parameter can be a proper list. But there's no way you would have been aware of that!

Reply:
Okay, done.

Since the list would be a separate parameter I also renamed this bool-switch to `AllowAlternateShell`, which seems more natural.